backend/src/graphql/resolver/BalanceResolver.ts

```diff
@@ -15,6 +15,8 @@ import { calculateDecay } from '@/util/decay'
 import { RIGHTS } from '@/auth/RIGHTS'
 import { GdtResolver } from './GdtResolver'
 
+import { getLastTransaction } from './accessLayer/getLastTransaction'
```

Comment:
please use the utils folder - making new folders without the proper refactor is making our code base impossible to maintain

Reply:
I do not like the repository construction. I wanted to place more stuff in the new folder in further refactoring

Reply:
The repository construct is the one we have + the utils folder. I don't think its wise to make a third structure within this PR.

Reply:
[This](https://medium.com/the-crowdlinker-chronicle/best-way-to-inject-repositories-using-typeorm-nestjs-e134c3dbf53c) seems like an interesting read to possibly reduce
```
import { UserRepository } from '@repository/User'
const userRepository = getCustomRepository(UserRepository)
const someValue = await userRepository.someQuery({..})
```
to 
```
const someValue = await userEntity.someQuery({..})
```

But hey, i dont know

Reply:
Read more here:

![image](https://user-images.githubusercontent.com/1238238/217118413-a6943a05-9715-4387-99e7-4efc1157f0e7.png)

https://stackoverflow.com/questions/71557301/how-to-workraound-this-typeorm-error-entityrepository-is-deprecated-use-repo

I think it is to summarize: TypeORM seems to have detected its shitty design and now might offer ways to overcome shortcommings.
I strongly believe we will be better off using their mechanics then our custom stuff.